test/view_models/channels_viewmodel_test.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:hng/models/app_model.dart';
+import 'package:hng/models/channel_members.dart';
+import 'package:hng/ui/view/channel/channel_list/channels_viewmodel.dart';
+import '../helpers/test_helpers.dart';
+
+// This is a test setup
+void main() {
```

Comment:
Please write the right tests for the channels view model not the CHANNELS LIST view model

Reply:
Check the name of the folder, it is channel view model. The person that writes the logic for the channels view model wrote it as "CHANNELS LIST view model".

Reply:
I wrote unit test for the right file which is "channels view model"